.changeset/nervous-hornets-reply.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
@amirabbas-gh can you remove this file? we don't need use changeset for the `latest` channel. only legacy rn